test/fixtures/dependencies.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 const path = require('path');
 const android = require('./android');
 
-const pjson = fs.readFileSync(path.join(__dirname, './files/dependency.json'));
+const pjson = fs.readFileSync(path.join(__dirname, 'files', 'package.json'));
```

Comment:
As I said - I'd better keep that package.json related to `dependencies` w/o any extra plugins included. What I liked in my PR is that we were defining these plugins on per suite basis with simple `mock` which was way more explicit and allowed us to test out more cases (duplicates, both dev & production deps, unique names exported). But just saying :package: